### DISCUSSION TITLE
Connections not persistent

### DISCUSSION THREAD

**Discussion by Karl-Goethebier** [blinksh/blink#2059] [Support]
My mosh connections persist only when i use LTE.

**Comment by carloscabanero** [comment]
Hey Karl! Thanks for reaching out. Could you send a few steps or what you are doing to try to replicate this situation? For example, you start a mosh connection over LTE, then you move to WiFi and it doesn't work? Is it always to the same WiFi? What happens if you start from WiFi and then move to LTE?

Most probably, what I'm looking here, is that you may be connecting over IPv6 in one network, say LTE, and then move to IPv4 on WiFi. If that's the case, please try to use the -4 flag, which would force IPv4 and see if the issue still happens.

**Reply by Karl-Goethebier** [reply]
--inet4, ja? Then i get this “Address Family can only be used with 'local' IP resolution (-R).
Use mosh1 for the deprecated (previous) mosh version.”

**Reply by Karl-Goethebier** [reply]
In other words: How should I do that? Or how does that work? Thanks in advance for any tips. An example would be great.
